### PR TITLE
Fixed column width computation for invisible series

### PR DESCRIPTION
**Pull request by chaosmail**
One more.

There was a small problem; the column width was not computed properly with invisible series. You can see the effect here [http://codepen.io/anon/pen/eNGxNG](http://codepen.io/anon/pen/eNGxNG) when you click on the legend to toggle the series. the column width stays the same.

Here you can see the fix in action: [http://codepen.io/anon/pen/LVzqVg](http://codepen.io/anon/pen/LVzqVg) now the column width is adapted properly.

I also extended a test to catch this.

Best,
Christoph

### CODE REVIEW

**Comment by chaosmail** [comment]
Hmmmm, such a weird error in Travis `Error: npm license has expired : uri-path`

File: src/utils/columns.coffee [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
-            if series.id? and series.id in stack.series
-              pseudoColumns[series.id] = index
-              keys.push(index) unless index in keys
-              inAStack = true
-
-          if inAStack is false
-            i = pseudoColumns[series.id] = index = keys.length
-            keys.push(i)
+        data.forEach (series, i) ->
+          visible = if options.series then options.series[i].visible else undefined
```
Comment (lorem--ipsum) [inline src/utils/columns.coffee]: This could just be `options.series?[i].visible`

**Comment by chaosmail** [comment]
Almost forgot this one; here is the update

**Comment by lorem--ipsum** [comment]
:poultry_leg:

**Pull request merged**